policyengine_uk/variables/gov/hmrc/tax_free_childcare/conditions/childcare_work_condition.py

```diff
@@ -0,0 +1,50 @@
+from policyengine_uk.model_api import * 
+
+class childcare_work_condition(Variable):
+    value_type = bool
+    entity = Person
```

Comment:
Similar question for each of the files - do we want to compute this for each person or for the entire Benefit Unit?